R/pkg/R/schema.R

```diff
@@ -29,11 +29,8 @@
 #' @export
 #' @examples
 #'\dontrun{
-#' sc <- sparkR.init()
-#' sqlCtx <- sparkRSQL.init(sc)
-#' rdd <- lapply(parallelize(sc, 1:10), function(x) { list(x, as.character(x)) })
 #' schema <- structType(structField("a", "integer"), structField("b", "string"))
-#' df <- createDataFrame(sqlCtx, rdd, schema)
+#' df <- createDataFrame(rdd, schema)
```

Comment:
nit: there is no `rdd` defined here. We can just use `createDataFrame(iris, schema)` or `dapply` for this example